repose-aggregator/components/filters/saml-policy-translation-filter/src/main/resources/META-INF/schema/config/saml-policy.xsd

```diff
@@ -0,0 +1,108 @@
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+           xmlns="http://docs.openrepose.org/repose/samlpolicy/v1.0"
+           targetNamespace="http://docs.openrepose.org/repose/samlpolicy/v1.0"
+           elementFormDefault="qualified"
+           attributeFormDefault="unqualified">
+
+    <!-- Elements -->
+    <xs:element name="saml-policy" type="SamlPolicyConfig"/>
+
+    <!-- Types -->
+    <xs:complexType name="SamlPolicyConfig">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>The setting to use for saml policy based translation.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:sequence>
+            <xs:element name="keystone-credentials" type="KeystoneCredentials" minOccurs="0"/>
+            <xs:element name="policy-cache" type="PolicyCache"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="KeystoneCredentials">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Keystone credentials to use if loading the policy is an authenticated call.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:attribute name="uri" type="xs:anyURI" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Rackspace Keystone v2 Identity Endpoint URI</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="username" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Username to access Keystone V2 identity service.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="password" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>User password to access Keystone V2 identity service.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="connection-pool-id" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Http Connection pool ID to use when talking to Keystone V2 Identity</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="PolicyCache">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Cache related settings</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:attribute name="ttl" type="xs:positiveInteger" use="required">
```

Comment:
Making this TTL a direct attribute deviates from the others in Repose. However since there is only one here and it is required, this probably makes more sense than having a single sub-element.